src/modules/facility-loan/facility-loan.service.get-loans-for-facility.test.ts

```diff
@@ -26,6 +27,7 @@ describe('FacilityLoanService', () => {
   let service: FacilityLoanService;
   let acbsFacilityLoanService: AcbsFacilityLoanService;
   let acbsBundleInformationService: AcbsBundleInformationService;
+  let repaymentScheduleBuilder: RepaymentScheduleBuilder;
```

Comment:
Similar to above, this variable is always undefined - maybe we should just use `null` in the constructor instead?